decapode/kafka/consumer.py

```diff
@@ -0,0 +1,54 @@
+import json
+from kafka import KafkaConsumer
+import logging
+
+from decapode import context
+from decapode.config import KAFKA_URI, KAFKA_API_VERSION
+
+log = logging.getLogger("decapode-kafka")
+
+
+async def consume_kafka_datasets() -> None:
+    '''For each message received in Kafka topic dataset, set all resources of the dataset to
+    priority for crawling.
+    '''
+    kafka_api_version = tuple([int(value) for value in KAFKA_API_VERSION.split('.')])
+    consumer = KafkaConsumer(
+        'resource.created',
+        'resource.modified',
+        'resource.deleted',
+        bootstrap_servers=KAFKA_URI,
+        group_id=None,
```

Comment:
We should have a group_id, shouldn't we?

Reply:
Correct, I'll add one